README.md

```diff
@@ -1,2 +1,152 @@
-# bookstoreredux
-edux is meant to help you to manage the state of your app globally. In this lesson you will learn how to add it to your project, using Redux Toolkit.
+# Books
+<a name="readme-top"></a>
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🛠 Tech Stack](#tech-stack)
+
+- [💻 Getting Started](#getting-started)
+  - [Install](#install)
+  - [Usage](#usage))
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+##  📖 books: setup project<a name="about-project"></a>
+
+**Bookstore** is a front-end application which can be used as a  hbook library.
+
+
+## 🛠 Built With
+ <a name="built-with">React</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://developer.mozilla.org/ru/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+
+### Key Features <a name="key-features">
+
+- A single page front-end application.
+- Book library
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+- Open your Terminal in your VSCode.
+- Navigate to the desired directory [folder] to which you want to clone the repository to
+- [Copy & Paste] Git clone you find in the code button https://github.com/samuelkinyanjui/bookstoreredux
+
+### Prerequisites
+
+In order to run this project you need:
+
+- General Understanding of JavaScript & CSS
+- basic Knowledge of Git Commands
+- basic Knowledge of the Terminal Commands
+
+###  📖 books: setup project/install<a name="about-project"></a>
+
+Clone this repository to your desired folder:
+
+- git clone https://github.com/samuelkinyanjui/bookstoreredux.git
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone https://github.com/samuelkinyanjui/bookstoreredux
+
+```
+
+### Install
+
+Must-Install-Project-with-the-Following-Programmes:
+
+- Prettier Extention in VScode
+- Git
+- Node.js
+- npm install```
+
+### Usage
+
+To run the project, execute the following command:
+
+> npm run build
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Kinyanjui Samuel**
+
+- GitHub: <a href="https://github.com/samuelkinyanjui">Samuel Kinyanjui </a>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- use Redux in React components.
+- connect to API
+- styling
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/samuelkinyanjui/bookstorereduxe/issues).
```

Comment:
- [ ] This particular link is currently broken as it throws up an error 404 page when it is clicked, It is unprofessional to leave broken links in your readme file so i kindly suggest you fix this and put the appropriate link :+1: :sparkles: